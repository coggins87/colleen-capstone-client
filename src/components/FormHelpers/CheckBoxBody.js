import React from 'react'

export const CheckBoxBody = props => {
    return (
      <li>
       <input key={props.id} onChange={props.handleCheck} type="checkbox" checked={props.isChecked} value={props.value} /> {props.value}
      </li>
    )
}

export default CheckBoxBody