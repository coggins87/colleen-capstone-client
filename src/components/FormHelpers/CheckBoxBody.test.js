import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CheckBoxBody from './CheckBoxBody'

describe(`CheckBoxBody component`, () => {
  const props = {
    className: 'test-class-name',
    testCheck: ()=>{},
    'data-other': 'test-other-prop'
  }

  it('renders a CheckBoxBody by default', () => {
    const wrapper = shallow(<CheckBoxBody />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the CheckBoxBody given props', () => {
    const wrapper = shallow(<CheckBoxBody {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})