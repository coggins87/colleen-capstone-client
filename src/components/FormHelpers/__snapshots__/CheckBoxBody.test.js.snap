// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBoxBody component renders a CheckBoxBody by default 1`] = `
<li
  className="checkbox_list"
>
  <input
    type="checkbox"
  />
   
</li>
`;

exports[`CheckBoxBody component renders the CheckBoxBody given props 1`] = `
<li
  className="checkbox_list"
>
  <input
    type="checkbox"
  />
   
</li>
`;
